error: ‘baseSize’ was not declared in this scope

I get the error 
```
SierraBreeze/breezedecoration.cpp:560:16: error: ‘baseSize’ was not declared in this scope
make[2]: *** [CMakeFiles/sierrabreeze.dir/build.make:111: CMakeFiles/sierrabreeze.dir/breezedecoration.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:388: CMakeFiles/sierrabreeze.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

when trying to compile it with recent commits.

Although I dont think it's the compiler, I'm using gcc 7.3

--- a/breezedecoration.cpp
+++ b/breezedecoration.cpp
@@ -555,6 +555,7 @@ namespace SierraBreeze
     //________________________________________________________________
     int Decoration::buttonHeight() const
     {
+        const int baseSize = settings()->gridUnit();
         // const int modifier = m_internalSettings->buttonRadius();
         const int modifier = m_internalSettings->buttonSize();
         return baseSize + modifier;

